### ISSUE TITLE
internal error: no model for initializing the net

### ISSUE THREAD

**Issue by vadamoto** [beniz/deepdetect#107]
Hello, I created a service "net9", trained it, and the model directory has the following files:

```
$ find /ddroot/net9_model -type f | xargs ls -lsa
       8 -rw-r--r-- 1 dd dd        4415 Apr 27 14:25 /ddroot/net9_model/convnet.prototxt
       4 -rw-r--r-- 1 dd dd         216 Apr 27 14:25 /ddroot/net9_model/convnet_solver.prototxt
       4 -rw-r--r-- 1 dd dd          63 Apr 27 11:57 /ddroot/net9_model/corresp.txt
       4 -rw-r--r-- 1 dd dd        3797 Apr 27 14:25 /ddroot/net9_model/deploy.prototxt
       4 -rw-r--r-- 1 dd dd        2075 Apr 27 14:25 /ddroot/net9_model/model.json
10704684 -rw-r--r-- 1 dd dd 10961592320 Apr 27 12:00 /ddroot/net9_model/train.lmdb/data.mdb
       4 -rw-r--r-- 1 dd dd        8192 Apr 27 12:00 /ddroot/net9_model/train.lmdb/lock.mdb
       0 -rw-r--r-- 1 dd dd           0 Apr 27 11:57 /ddroot/net9_model/vocab.dat
```

restarted the container and created "net9" again

```
2016-04-27 17:25:05.193391 PUT /services/net9
{"model": {"templates": "../templates/caffe/", "repository": "/ddroot/net9_model"}, "type": "supervised", "description": "character-based classifier", "parameters": {"input": {"connector": "txt", "alphabet": "abcdefghijklmnopqrstuvwxyz0123456789,;.!?'\"#\"/\\|_@#$%^&*~`+-=<>\u0430\u0431\u0432\u0433\u0434\u0435\u0436\u0437\u0438\u0439\u043a\u043b\u043c\u043d\u043e\u043f\u0440\u0441\u0442\u0443\u0444\u0445\u0446\u0447\u0448\u0449\u044a\u044b\u044c\u044d\u044e\u044f\u0451", "sequence": 140, "characters": true, "sentences": true}, "mllib": {"layers": ["1CR256", "1CR256", "4CR256", "1024", "1024"], "dropout": 0.5, "db": true, "nclasses": 9, "template": "convnet"}, "output": {}}, "mllib": "caffe"}
2016-04-27 17:25:05.224457 {"status":{"code":201,"msg":"Created"}}
```

Now I'm running

`curl -X POST 'http://localhost:8080/predict' -d '{"data": ["/ddroot/test_file.net9.utf.txt"], "parameters": {"input": {"sentences": true}, "mllib": {"gpu": true}, "output": {"measure": ["f1"]}}, "service": "net9"}'`

and getting

`{u'status': {u'msg': u'InternalError', u'code': 500}}`

while the program says

`service net9 mllib internal error: no model in /ddroot/net9_model for initializing the net`

**Comment by beniz** [comment]
Hi, your training certainly has failed or was misplaced because your repository does not hold any trained network weight file (must end with `.caffemodel`), as the server is telling you. On the side note you appear to have not `test.lmdb` repository either, which indicates that for some reasons you have not monitored the performance of your model.

To debug your issue, I'd start verifying that your training step did actually end without error.

**Comment by vadamoto** [comment]
Thanks, now it's more clear to me :)
I was running it with --tsplit=0 which seemed right at the moment, since I had another dataset for testing. I ran again with default tsplit, and it took much-much longer :)

**Comment by beniz** [comment]
If you have a test set you should be able to pass it on as the second argument in the `data` array. Let me know if you have difficulties.

**Comment by beniz** [comment]
Over 15 days without activity, closing.
